openghg/store/_boundary_conditions.py

```diff
@@ -66,7 +71,7 @@ def read_file(
         filters: Optional[Any] = None,
         chunks: Optional[Dict] = None,
         optional_metadata: Optional[Dict] = None,
-    ) -> dict:
+    ) -> Dict:
```

Comment:
Question for @brendan-m-murphy on this - were we aiming to move to `dict` rather than using `Dict`? In which case we should keep this as `dict`?
```suggestion
    ) -> dict:
```

Reply:
`dict` is now preferred, but we'll probably use a linter to update the type hints now that we have 3.10 (e.g. before the next release, just update everything)